models/issues/issue.go

```diff
@@ -1273,7 +1273,9 @@ func (opts *IssuesOptions) setupSessionNoLimit(sess *xorm.Session) {
 		applySubscribedCondition(sess, opts.SubscriberID)
 	}
 
-	if len(opts.MilestoneIDs) > 0 {
+	if len(opts.MilestoneIDs) == 1 && opts.MilestoneIDs[0] == db.NoConditionID {
+		sess.And("issue.milestone_id = 0")
```

Comment:
The name `NoConditionID` looks really strange. When reading the code:

1. The parameter is no-condition-ID
2. The the next line adds a condition .....

Just a nit, just my opinion, not blocker.